Bottom: 2e2c0eada4600f94128c5baf2fa3a5ae9cece349
Top:    4d63719c20f02301c2047a14c563f106bb1e583b
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-08 01:00:31 +0100

drc: HACK HACK HACK


---

diff --git a/src/find.c b/src/find.c
index 6edd29d..4724169 100644
--- a/src/find.c
+++ b/src/find.c
@@ -2623,7 +2623,12 @@ IsPolygonInPolygon (PolygonTypePtr P1, PolygonTypePtr P2)
 
               line.Point1.X = v->point[0];
               line.Point1.Y = v->point[1];
-              line.Thickness = 2 * Bloat;
+              line.Thickness = Bloat;
+              /* Another Bloat is added by IsLineInPolygon, making the correct
+               * 2x Bloat. Ideally we would change it there, but doing so
+               * breaks some other DRC checks which rely on the broken behaviour
+               * in IsLineInPolygon.
+               */
               line.Clearance = 0;
               line.Flags = NoFlags ();
               for (v = v->next; v != &c->head; v = v->next)
